tools/luh2/luh2.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+# WARNING: This script generates intermediate copies of the LUH2
+# data which at its peak takes up approximately 42G of space.
+#
+# Note that this script must be run with the luh2 conda environment
+# It requires a single argument that points to the full path location
+# of the luh2 data and the dataset to regrid against
+
+# LUH2 data names
+DATA_LOC=$1
+STATIC_LOC=$2
+TARGET_LOC=$3
+OUTPUT_LOC=$4
+STATES_FILE=states.nc
+TRANSITIONS_FILE=transitions.nc
+MANAGE_FILE=management.nc
```

Comment:
@glemieux and I discussed the risk of using the same three file names (states, transitions, management) for all cases that are present in /glade/p/cesmdata/cseg/inputdata/lnd/clm2/rawdata/LUH2. The most likely risk that comes to mind is one where we copy or move files without keeping track of the contents. We looked in a few files for metadata that may help differentiate between files but did not find enough information.

@glemieux and I recommend renaming the files according to case/scenario/version/other identifying characteristics. Doing so requires modifications to the tool, but maybe just in these few lines?

I suggested that @glemieux move forward without changing the file names for now. When the tool gets added to Erik's mksurfdata_esmf makefile, we can determine whether changing the file names adds significant complication. We may also decide to keep the file names unchanged in order to maintain "purity" relative to the original source of the data on the LUH2 website.

Reply:
Converted the comment to #1058 for future discussion.